build/configs/rtl8720e/hello/Make.defs

```diff
@@ -0,0 +1,228 @@
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
Please check and update the year to 2022.

Reply:
Thank you! I have checked and update the rest files